### PR TITLE
test: add text compliance tests

### PR DESCRIPTION
**Pull request by HemangChothani**

### CODE REVIEW

**Comment by HemangChothani** [comment]
Failed tests are fixed in https://github.com/cloudspannerecosystem/python-spanner-sqlalchemy/pull/23 PR.

File: test/test_suite.py [hunk 1]
```diff
@@ -43,7 +43,7 @@
 from sqlalchemy import LargeBinary
 from sqlalchemy import Boolean
 from sqlalchemy import String
-from sqlalchemy.types import Integer
+from sqlalchemy.types import Integer, Text
 from sqlalchemy.types import Numeric
```
Comment (vi3k6i5) [inline test/test_suite.py:47]: nit: can merge this also above if merging multiple, else keeping each separate is okay to be consistent across files.

File: test/test_suite.py [hunk 2]
```diff
@@ -837,3 +838,72 @@ def test_nolength_binary(self):
...
+
+class TextTest(_TextTest):
+    @pytest.mark.skip("Spanner throws an error")
+    def test_text_empty_strings(self, connection):
+        """
+        SPANNER OVERRIDE:
+
+        Spanner DBAPI throws an error when cleanup tried to
+        rollback the connection after the test executed successfully.
+        The error is `ValueError: Transaction is already rolled back`.
```
Comment (vi3k6i5) [inline test/test_suite.py:851]: would it be better to either skip the roll back by mocking it or overriding that part. The original test case seem valid and fine: https://github.com/sqlalchemy/sqlalchemy/blob/ea225c63f98dc3c250cee1b243762970b8e0932a/lib/sqlalchemy/testing/suite/test_types.py#L219

**Comment by HemangChothani** [comment]
One test and cleanup of two tests are failing which will be resolved by respective PRs https://github.com/cloudspannerecosystem/python-spanner-sqlalchemy/pull/23, https://github.com/cloudspannerecosystem/python-spanner-sqlalchemy/pull/68.

**Review by larkee — APPROVED** [review]
LGTM :+1:

**Pull request merged**